### DISCUSSION TITLE
Enforce log in

### DISCUSSION THREAD

**Discussion by jameswoodley** [OrchardCMS/OrchardCore#11030] [Q&A]
I have just enabled Google login on our instance, and I'd like to enforce all users go through this path. Is it possible to log out all current users and enforce them to re-login through Google?

**Comment by ns8482e** [comment]
I guess disabling local login will do the trick

**Reply by jameswoodley** [reply]
Will users already signed in, be logged out?

**Comment by MichaelPetrinolis** [comment]
Hi, if only one external provider is configured (Google for your case), you should enable the 'Use external provider for login' (probably you have already configured the registration options) and the 'Disable local password login'.  You can also reset all the users' password to null in a migration so that the local accounts can't login with local password. Perform your tests on a dev environment to check the available options.

**Reply by jameswoodley** [reply]
Will users already signed in, be logged out?

**Comment by MichaelPetrinolis** [comment]
OC uses Microsoft Identity for authentication. Check if it provides a way to invalidate the cookie per user. 
Then in a migration step you could perform this action on all the users without external linked account.

**Comment by ns8482e** [comment]
When ever you make identity change to invalidate persistence cookie you need sign out user with your custom logic,   You can create workflow activity that forcefully logout
